defaults/main.yml

```diff
@@ -1,5 +1,14 @@
 ---
-ntp_enabled: true
+incompatible_vm_types:
+  - openvz
+  - docker
+
+ntp_enabled: |-
```

Comment:
Can you put in a comment here that mentions the logic / reasoning? Just so it's clear to users of the role who look in this file. Also it might be good to update the README to mention that by default, the role will set `ntp_enabled` to `false` if it detects certain types of virtualized environments.